blog/views.py

```diff
@@ -1,4 +1,62 @@
 """Views of blog app."""
-# om django.shortcuts import render
+from django.utils import timezone
+from django.shortcuts import render, redirect, get_object_or_404
+from django.contrib.auth.decorators import login_required
 
-# Create your views here.
+from .models import Post
+from .forms import PostForm
+
+
+def post_list(request):
+    """Classic Django posts list view."""
+    posts_list = Post.objects \
+                     .filter(published_date__lte=timezone.now()) \
+                     .order_by('-published_date')
```

Comment:
Using EOL escaping may lead to unobvious bugs.

For example, this is a valid code example:
```python
s = 'some'\
'string'
```
And this is too:
```python
s = 'some'\

'string'
```
But the difference is that accidental extra newline separates the statements and `'string'` becomes it's own separate statement and not the part of one above it.

Better wrap things with braces and this problem will disappear:
```python
s = (
    'some'
    'string'
)
s = (
    'some'

    'string'
)
```